pandas/tests/series/indexing/test_indexing.py

```diff
@@ -907,3 +907,18 @@ def test_head_tail(test_data):
     assert_series_equal(test_data.series.head(0), test_data.series[0:0])
     assert_series_equal(test_data.series.tail(), test_data.series[-5:])
     assert_series_equal(test_data.series.tail(0), test_data.series[0:0])
+
+
+def test_index_map():
```

Comment:
This should probably go in `pandas/tests/indexes/datetimes/test_datetime.py` with the other `test_map` functions.